umbraco-cloud/set-up/project-settings/manage-hostnames/rewrites-on-cloud.md

```diff
@@ -56,17 +55,17 @@ Once you've applied a certificate to your site, you can make sure that anybody v
 
 To accomplish this, add a rewrite rule to the live environment's `web.config` in the `<system.webServer><rewrite><rules>` section.
 
-For example, the following rule will redirect all requests for the site http://mysite.com URL to the secure https://mysite.com URL and respond with a permanent redirect status.
+For example, the following rule will redirect all requests for the site http://example.com URL to the secure https://example.com URL and respond with a permanent redirect status.
 
 ```xml
-<rule name="HTTP to HTTPS redirect" stopProcessing="true">
-  <match url="(.*)" />
+<rule name="Redirect HTTP to HTTPS" stopProcessing="true">
```

Comment:
We could also add an (easier) alternative that uses the HTTPS Redirection Middleware and optionally also adds HSTS in the application code instead: https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl#usehttpsredirection.

Reply:
Could it maybe make sense to have both versions? 

The one we already have and then one that is using the middleware HTTPS redirection?

Reply:
Documenting both versions would make sense yes and looking at the CMS docs, this is already done in https://docs.umbraco.com/umbraco-cms/reference/security/ssl-https. So I would recommend replacing this section with a link to the general CMS instructions 👍🏻

Reply:
Perfect! 

I will add a task for us to replace it with a link to the CMS documentation :D